packages/components/inputs/localized-rich-text-input/src/editor.tsx

```diff
@@ -17,6 +26,7 @@ import { messagesMultilineInput } from '@commercetools-uikit/input-utils';
 import {
   RichTextBody,
   HiddenInput,
+  type TEditor,
```

Comment:
following the comment from Carlos I guess [this approach](https://github.com/commercetools/ui-kit/pull/2129#discussion_r814795761) will make us less coupled with `rich-text-utils` later on

Reply:
I've implemented the approach. Thanks.